### PR TITLE
Change IMU timestamp field to us since boot

### PR DESCRIPTION
**Pull request by dpkoch**
This is linked to BYU-MAGICC/ROSflight2#34

### CODE REVIEW

File: message_definitions/v1.0/rosflight.xml [hunk 1]
```diff
@@ -44,20 +44,20 @@
             <field type="int16_t" name="value4">Fourth value, interpreted according to mode</field>
         </message>
         <message id="181" name="SMALL_IMU">
-            <field type="int16_t" name="stamp">Time in us since last IMU message</field>
+            <field type="int32_t" name="time_boot_us">Measurement timestamp as microseconds since boot</field>
```
Comment (superjax) [inline message_definitions/v1.0/rosflight.xml]: What if we made this a `uint32_t`
Also, what if we added a `int16_t` temperature field?

**Comment by superjax** [comment]
Ah!  you made a pull request of this. 

Let me know what you think about my comments.

**Comment by dpkoch** [comment]
@superjax Oh, yeah that was supposed to be a `uint32_t`, just a typo. I'll fix that.

So yeah we could add a temperature field. I suppose the only time that'll get used is for the temperature calibration? At other times the compensation will be done on board so we won't use that field on the ROS side I think. But there's probably not a good way to do the calibration otherwise, so are we okay with having that overhead to enable the calibration?

**Comment by superjax** [comment]
I guess the only alternative is to publish the temperature as a separate message.  I'm not entirely opposed to that either, especially if we can guarantee that the temperature comes before the IMU message on the mavlink side, then the ROS side should work just fine, with a shared temperature variable.  It just makes it more complicated.

I don't know.  I'm hesitant to add so much complexity for the sake of bandwidth.  (especially with the streaming rates we've been seeing lately)

**Pull request merged**